src/renderers/common/extras/PMREMGenerator.js

```diff
@@ -318,9 +318,7 @@ class PMREMGenerator {
 
 	_compileMaterial( material ) {
 
-		const tmpMesh = this._lodMeshes[ 0 ];
-		tmpMesh.material = material;
-
+		const tmpMesh = new Mesh( this._lodPlanes[ 0 ], material );
 		this._renderer.compile( tmpMesh, _flatCamera );
```

Comment:
`Renderer.compile()` is async. It's probably better to make the compile methods in `PMREMGenerator` as well.

Reply:
We should rename it to `PMREMGenerator.compileAsync()` too? or keep both maybe adding a warning in compile()?